content/docs/2.7/scalers/gcp-storage.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Google Cloud Platform Storage"
+availability = "2.7+"
+maintainer = "Community"
+description = "Scale applications based on the count of objects in a given GCP Storage bucket."
+layout = "scaler"
+go_file = "gcp_storage_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `gcp-storage` trigger for GCP Storage. It scales based on the count of objects in a given GCS bucket and assumes the worker is responsible for clearing the bucket by deleting/moving the objects once their processing is complete.
+
+```yaml
+triggers:
+- type: gcp-storage
+  metadata:
+    bucketName: test-bucket
+    targetObjectCount: '100'
+    maxBucketItemsToScan: '1000'
+    credentialsFromEnv: GOOGLE_APPLICATION_CREDENTIALS_JSON # Optional
+    credentialsFromEnvFile: GOOGLE_APPLICATION_CREDENTIALS_JSON # Optional
+```
+
+**Parameter list:**
+
+- `bucketName` - Name of the bucket in GCP Storage.
```

Comment:
```suggestion
- `bucketName` - Name of the bucket in GCS.
```

Reply:
Not everyone knows what GCS stands for though, but one could assume people should know when they use that service.

Still, might be good to not use abbreviations other than GCP for example